src/components/Field.jsx

```diff
@@ -66,13 +66,6 @@ const Field = props => {
 export default Field
 
 class FieldWrapperComponent extends Component {
-  componentDidMount() {
-    if (this.props.giveFocus)
-      ReactDOM.findDOMNode(this)
-        .querySelector('input')
-        .focus()
-  }
-
```

Comment:
Here we lose the giveFocus feature? Why not simply check for the querySelector result ?